### DISCUSSION TITLE
Purpose of row-cols?

### DISCUSSION THREAD

**Discussion by ricktig** [twbs/bootstrap#34615] [Q&A]
I placed this code copied from the Bootstrap 5 documentation into a simple project thinking that I'm going to get the 2x2 grid I'm looking to build.  Instead, I get four columns across with the word Column in each.  It appears that row-cols didn't do anything so I'm confused as to its purpose.

                            <div class="row row-cols-2">
                                <div class="col">Column</div>
                                <div class="col">Column</div>
                                <div class="col">Column</div>
                                <div class="col">Column</div>
                              </div>

**Comment by alpadev** [comment]
It does exactly what is explained here https://getbootstrap.com/docs/5.0/layout/grid/#row-columns.
The example in the documentation uses the same code and renders a 2x2 layout (in my case).

And here is a standalone example that behaves as expected.
https://jsfiddle.net/am51e9j8/

Have you included the CSS file https://getbootstrap.com/docs/5.0/getting-started/introduction/#css?
